### DISCUSSION TITLE
Help with Error message: Publii is unable to find remote files list on your server.

### DISCUSSION THREAD

**Discussion by plasmidone** [GetPublii/Publii#1752] [Help] [answered]
Hello. I already have one Publii site that syncs fine with GitLab. Yesterday, I created a second GitLab account and Publii website which is a clone of the first website. The settings are all the same / similar but yet I get an error message: 

"Publii is unable to find remote files list on your server. If you will continue the sync process, ALL generated website files will be uploaded to your server. You can cancel the sync process to check the reason of your issue and then try again."

If I proceed and select "Continue and sync", I get the following error: 

"An error occurred while connecting to the server: Response code 400 (Bad Request)".

I've been searching for a solution the whole day yesterday and today and I've gone through every settings page of GitLab to make sure everything is setup the exact same way as the previous website. Any help would be appreciated! Thanks

<img width="1065" alt="Screenshot 2024-10-12 at 15 09 10" src="https://github.com/user-attachments/assets/f24df8d1-3e08-4e31-8e18-02a72ebcf57b">
<img width="1031" alt="Screenshot 2024-10-12 at 17 11 34" src="https://github.com/user-attachments/assets/03346113-fb2d-4c8e-97a0-58187a65e5c0">

**Comment by plasmidone** [comment]
<img width="1552" alt="Screenshot 2024-10-13 at 16 12 21" src="https://github.com/user-attachments/assets/fc4b702c-fb2d-48c3-a3dd-05e5d71f21fd">
<img width="623" alt="Screenshot 2024-10-13 at 16 17 15" src="https://github.com/user-attachments/assets/d1ab4e20-dc61-401e-b0eb-6ef57eb8d74e">
<img width="756" alt="Screenshot 2024-10-13 at 17 00 45" src="https://github.com/user-attachments/assets/aad9f089-c69a-41dd-bcac-c7fe67ac5fbb">

I keep searching for a solution, I also get an error 520 if I proceed with the sync and try again. I tried to create a new blank project in the same account and recreated a new ".gitlab-ci.yml" file. It fails to upload the files and I get notification emails telling me about a failed pipeline for "main". When I look that up, there are some discussions about verifying GitLab accounts with a credit card but there should be a banner telling me like below and I do not get that banner in my new account...

<img width="709" alt="Screenshot 2024-10-13 at 17 07 08" src="https://github.com/user-attachments/assets/b5587762-367a-4b45-9bd9-039edb9efc21">

**Comment by plasmidone** [comment]
I managed to find some log of the job and the following line shows an "uploader error" but I don't know the implications or what that means:
`WARNING: Upload request redirected                  location=https://gitlab.com/api/v4/jobs/8073012783/artifacts?artifact_format=zip&artifact_type=archive new-url=https://gitlab.com/
WARNING: Retrying...                                context=artifacts-uploader error=request redirected`

**Comment by plasmidone** [comment, marked as answer]
Just an update in case other's encounter the same issues. I gave up and stopped using GitLab. I started using Netlify and it was so much easier to setup and worked great straightaway. GitLab is not fit for purpose when using with Publii imo.

**Comment by dperarnaud** [comment]
I have the same problem with my OVH hosting.
For me, the problem is not gone.
We need to know if this is a problem generated by Publii or the hosting company.
